deploy/opencraft.yml

```diff
@@ -0,0 +1,11 @@
+- name: Deploy the OpenCraft Instance Manager
+  hosts: all
+  become: true
+  roles:
+    - name: common-server
+      COMMON_SERVER_NO_BACKUPS: true    # The instance manager VM is stateless
+      FORWARD_MAIL_MYDOMAIN: "{{ inventory_hostname }}"
```

Comment:
You'll need to add {{forward-server-mail}} role explicitly, the only way to fix the 20-30 mails sent during the initial provisioning run was to remove this role from dependencies of {{common-server}}  and add it to playbook after commmon server role. I thought about different solutions, but this one was least bad.